app/services/helpers/table-creator.js

```diff
@@ -30,3 +30,13 @@ module.exports.createCapacityTable = function (id, rowLabel, capacityDateRange,
 
   return new DisplayTable(headings, rows)
 }
+
+function customRound (number) {
```

Comment:
Some unit tests on this would be nice - Like what happens if the `number` is undefined?